enterprise_subsidy/apps/subsidy/models.py

```diff
@@ -227,22 +227,16 @@ def create_transaction(
             **transaction_metadata,
         )
 
-    def commit_transaction(self, ledger_transaction, reference_id=None, reference_type=None):
+    def commit_transaction(self, ledger_transaction, fulfillment_identifier=None):
```

Comment:
Don't we also need to accept an external_reference_id here and use it to create a new ExternalTransactionReference record?

Reply:
ahhhhhh yes! Excellent call out